### PR TITLE
Slim trajectories

### PR DESCRIPTION
**Pull request by yramachers**
A bit of a cleanup PR, removing redundant storage items, add to the readme, introduce a trial map as new output data column in the ntuple to code for logical volume names containing the vertex coordinates as unique integers, lookup is in the readme as are now the output column descriptions.

### CODE REVIEW

**Comment by yramachers** [comment]
Found the paper from the Biasing mechanism authors on arXiv (there is a NIM reference too) and noted that the main step in biasing XS is missing in this code - no application of the weight factor. This is however necessary in order for the biasing to work correctly. My excuse is that they don't use it either in their example GB01 but I should have considered this earlier. 
The technique to get the correct weight factor is fortunately very simple - the G4Track collects the accumulated weight, i.e. taking account of any XS biasing in a process agnostic way, i.e. we don't have to worry about biasing two processes which might take place several times in any event.
I should therefore add an additional primitive scorer which records the G4Track->GetWeight() for every hit in the sensitive detector. That should allow to check swiftly whether the output makes sense and allows to have biased hit counting as well the weight correction available for analysis post-processing.
The plan is submit this new feature in this PR, as a replacement for the removed ParentID scorer.

**Comment by yramachers** [comment]
With the new weight scorer it also becomes clear that we can't arbitrarily ramp up the bias factors. A scoring with factor 100 on one process and 1000 on the other already results in extremely low weights. Leaving n,gamma as is, with the hard-coded 68% increase, and boosting muNuclear reactions by 100 however, gives reasonable weights in the percent level on small sample tests.

File: src/WLGDPSTrackWeight.cc [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+void WLGDPSTrackWeight::PrintAll()
+{
+  G4cout << " MultiFunctionalDet  " << detector->GetName() << G4endl;
+  G4cout << " PrimitiveScorer " << GetName() << G4endl;
+  G4cout << " Number of entries " << EvtMap->entries() << G4endl;
+  auto itr = EvtMap->GetMap()->begin();
+  for(; itr != EvtMap->GetMap()->end(); itr++)
+  {
+    G4cout << "  key: " << itr->first << "  total weight: " << *(itr->second) << G4endl;
+  }
```
Comment (drbenmorgan) [inline src/WLGDPSTrackWeight.cc:65]:
```suggestion
  for(const auto& itr : *(EvtMap->GetMap()))
  {
    G4cout << "  key: " << itr.first << "  total weight: " << *(itr.second) << G4endl;
  }
```

Comment (drbenmorgan) [reply src/WLGDPSTrackWeight.cc:65]: Could also try out C++17 structured bindings here - I _think_ the following works:

```
for (const auto& [k, v] : *(EvtMap->GetMap()))
{
      G4cout << "  key: " << k << "  total weight: " << *v << G4endl;
}
```

**Comment by yramachers** [comment]
As the next branch I would like to move to the production version, i.e. change the hit creation to Ge77 production only, removing the neutrons scoring in germanium.
Can you think of any reason not to take that step yet? Did I forget anything?

**Comment by yramachers** [comment]
Bumping this up for review @drbenmorgan

**Pull request merged**